BennyTordrupVisma/Week3/Ass4/dist/bundle.js

```diff
@@ -0,0 +1,485 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Usually it is a bad idea to check-in the bundle file. It would be the same as adding the bin/obj folders to git in a .NET project. `bundle.js` have been added globally to gitignore to I think you might just have committed it before the gitignore was updated